pkg/statistics/utils/constant.go

```diff
@@ -12,17 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package statistics
+package utils
 
 import (
-	"fmt"
+	"github.com/docker/go-units"
 )
 
 const (
-	// StoreHeartBeatReportInterval is the heartbeat report interval of a store.
-	StoreHeartBeatReportInterval = 10
-	// RegionHeartBeatReportInterval is the heartbeat report interval of a region.
-	RegionHeartBeatReportInterval = 60
+	// WriteReportInterval indicates the interval between write interval, the value is the heartbeat report interval of a region.
+	WriteReportInterval = 60
+	// ReadReportInterval indicates the interval between read stats report, the value is the heartbeat report interval of a store.
+	ReadReportInterval = 10
```

Comment:
How about still using StoreHeartBeatReportInterval and RegionHeartBeatReportInterval?

Reply:
Do you mean removing WriteReportInterval?

Reply:
yes